[docs][base] Add pages for coming soon components (#37575)
mui/material-ui: docs/data/base/components/accordion/accordion.md

---

> product: base

Reviewer: Since [this PR](https://github.com/mui/material-ui/pull/37593) has merged, you'll have to use `base-ui` instead of `base`

Author: @brijeshb42 Do I need to update all of them now? Or can I do it on a separate PR?

Author: Asking as I thought only these new pages I added were outdated but realized all of them are using `base` instead of `base-ui`, so not sure if doing it right now is the best moment. No worries about doing it though ⎯ just making sure!

Reviewer: Have you rebased your changes with latest master? All md files are now using `base-ui` only.

Author: Ah perfect ⎯ just did that. Thanks for the heads-up 🤙

title: React Accordion component
githubLabel: 'component: accordion'
waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
---

# Accordion

<p class="description">Accordions let users show and hide sections of related content on a page.</p>

:::info
💡 The Base UI Accordion component is still in development.

> If you're in need of it, please give a shout about your need of it in [**this issue**](https://github.com/mui/material-ui/issues/27170) to help us prioritize the next batch of new components.

Reviewer 2: > If you're in **need of it**, please give a shout about your **need of it**

That's a bit repetitive, could you somehow rephrase it?

Author: Let me tag @samuelsycamore for the copywriting magic over here 😬

Author: How about:

> If you're in need of it, please give us a shout about it in this issue to help us prioritize the next batch of new components.

What do y'all think?

Reviewer 3: For something like this I'd keep it short and direct:

```suggestion
Help us prioritize the next batch of new components by sharing your needs in [this GitHub issue](https://github.com/mui/material-ui/issues/27170).
```

Author: Sweet — much better 👌

Reviewer 2: The text should also be updated in other components.

Author: 🤦 Done ⎯ thanks for pointing out!
